custom/templates/DefaultRevamp/forum/view_topic.tpl

```diff
@@ -14,7 +14,7 @@
   <div class="topic title">{if count($TOPIC_LABELS)}{foreach from=$TOPIC_LABELS item=label}{$label} {/foreach}{/if}{$TOPIC_TITLE}</div>
   <div class="sub header">
     {$started_by_val="<a href=\"`$TOPIC_AUTHOR_PROFILE`\" style=\"`$TOPIC_AUTHOR_STYLE`\">`$TOPIC_AUTHOR_USERNAME`</a>"}
-    {$STARTED_BY|replace:'{x}':$started_by_val}
+    {$STARTED_BY|replace:'{{author}}':$started_by_val}
```

Comment:
Any ideas what this is, or if we can do the replacement by i18n?

Reply:
Replaced!